### PR TITLE
fix

### PR DESCRIPTION
**Pull request by webdev03**
Signed-off-by: webdev03 <75148774+webdev03@users.noreply.github.com>

Should be fixed.

### CODE REVIEW

**Comment by hello-smile6** [comment]
I made my own ( #34 ). I'll close mine.

**Comment by webdev03** [comment]
@JaydenDev pls merge i tested it lol

**Comment by hello-smile6** [comment]
> @JaydenDev pls merge i tested it lol

Merging it into my fork so I can test.

**Comment by JaydenDev** [comment]
k

**Comment by hello-smile6** [comment]
> > @JaydenDev pls merge i tested it lol
> 
> Merging it into my fork so I can test.

https://github.com/hello-smile6/freecat/pull/1 merged it into mine.

File: app/main/main.js [hunk 1]
```diff
@@ -10,7 +10,8 @@ function createWindow () {
     webPreferences: {
       preload: path.join(__dirname, 'preload.js'),
       webviewTag: true
-    }
+    },
+    /* icon: path.join(__dirname, "../icons/freecat-2.icns") */
```
Comment (hello-smile6) [inline app/main/main.js:14]: Nice, could you add an icon for the rest of us?

**Review by hello-smile6 — CHANGES_REQUESTED** [review]
Could you finish the icon part?

File: app/main/main.js (hunk 1 above)
Comment (webdev03) [reply app/main/main.js:14]: I'm working on it, doesn't seem to work though...

Comment (hello-smile6) [reply app/main/main.js:14]: > 

Just use `<link rel="shortcut icon">`

**Review by webdev03 — COMMENTED** [review]
im kinda dumb for not noticing this earlier :sob:

File: app/main/main.js (hunk 1 above)
Comment (webdev03) [reply app/main/main.js:14]: Oh, okay, that works, I'll try it!

File: app/main/main.js [hunk 2]
```diff
@@ -43,7 +44,7 @@ function createWindow () {
 // This method will be called when Electron has finished
 // initialization and is ready to create browser windows.
 // Some APIs can only be used after this event occurs.
-app.whenReady().then(() => {
+app.on("ready", () => {
```
Comment (hello-smile6) [inline app/main/main.js:47]: Arrow notation! Nice!

File: app/main/main.js [hunk 3]
```diff
@@ -21,7 +22,7 @@ function createWindow () {
 //      click: () => {  }
 //    }]
 //  }));
-  app.whenReady().then(() => {
```
Comment (hello-smile6) [inline app/main/main.js:24]: It worked for me, wonder where that came from.

File: app/main/main.js [hunk 4]
```diff
@@ -21,7 +22,7 @@ function createWindow () {
...
+  app.on('ready', () => {
```
Comment (hello-smile6) [inline app/main/main.js:25]: Yay, more arrow notation!

File: app/main/main.js [hunk 5]
```diff
@@ -1,5 +1,5 @@
 // Modules to control application life and create native browser window
-const {app, BrowserWindow, Menu, MenuItem, globalShortcut, shell} = require('electron')
+const {app, BrowserWindow, /* Menu, MenuItem, */ globalShortcut, shell} = require('electron')
```
Comment (hello-smile6) [inline app/main/main.js:2]: Bye-bye, extra RAM usage!

File: app/main/main.js [hunk 6]
```diff
@@ -1,5 +1,5 @@
 // Modules to control application life and create native browser window
-const {app, BrowserWindow, Menu, MenuItem, globalShortcut, shell} = require('electron')
```
Comment (hello-smile6) [inline app/main/main.js:2]: That's a lot of imports, happy we can drop some!

**Review by hello-smile6 — COMMENTED** [review]
Yay! May ES6 arrow functions live forever!

**Comment by hello-smile6** [comment]
Why did you close it?

**Comment by webdev03** [comment]
Because there's a rickroll for some reason and nobody else is having it.

**Comment by webdev03** [comment]
Seems like it's from my computer or something, but I swear I didn't enter that in

**Comment by hello-smile6** [comment]
> Because there's a rickroll for some reason and nobody else is having it.

You mean, c3e7680b75d8c46d664da54b56c21dc4e43dc7fa?

**Comment by webdev03** [comment]
Oh that makes sense.. it was you.. but then why did you make a discussion if you knew what happened?

**Comment by hello-smile6** [comment]
> Oh that makes sense.. it was you.. but then why did you make a discussion if you knew what happened?

Seeing if people noticed. 🙂 @JaydenDev Actually pushed a release with that rickroll. I'll revert that commit, since it caused all these problems in the first place.

**Comment by webdev03** [comment]
Honestly, it's kind of cool, I guess it could be there for a bit if JaydenDev allows.. sorry for the misunderstanding

**Comment by hello-smile6** [comment]
> Honestly, it's kind of cool, I guess it could be there for a bit if JaydenDev allows.. sorry for the misunderstanding

No, it's disruptive. I don't want people to forget about it and get rickrolled. It's bad enough that a release contains that commit.

**Comment by JaydenDev** [comment]
finished?

**Pull request merged**